controllers/infra/Makefile

```diff
@@ -67,8 +67,7 @@ build:  ## Build manager binary.
 
 .PHONY: build-multi-arch
 build-multi-arch:  ## Build manager binary.
-	CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -o bin/controller-infra-amd64 main.go
-	CGO_ENABLED=0 GOOS=linux GOARCH=arm64 go build -o bin/controller-infra-arm64 main.go
+	CGO_ENABLED=0 GOOS=linux go build -o bin/manager main.go
```

Comment:
We need this for test purpose multi-arch docker-file build, so maain branch use keeped version of build for ci.